pom.xml

```diff
@@ -478,15 +478,16 @@
                 <scope>test</scope>
             </dependency>
 
+            <!-- Override vulnerable okhttp 3.14.1 used by jslack -->
             <dependency>
                 <groupId>com.squareup.okhttp3</groupId>
                 <artifactId>okhttp</artifactId>
-                <version>3.14.9</version>
+                <version>${okhttp.version}</version>
```

Comment:
I am not sure how it is solving the issue just by replacing <version>${okhttp.version}</version> to <version>3.14.9</version>, in my opinion both are same

Reply:
In fact the opposite is done here - 3.14.9 is replaced by `${okhttp.version}`. `okhttp.version` is defined in the properties section and set to 4.10.0. 

I combined these 2 actions into one because detected that `mockwebserver` dependency shares the version with `okhttp` - they both have 3.14.9. Upgrading just 1 dependency may introduce an inconsistency with `mockwebserver`. So, I decided
1.  to replace explicitly defined 3.14.9 version by `${okhttp.version}` property reference and
2. to set that property to newer non-vulnerable version - 4.10.0.